tensorboard/webapp/util/lang.ts

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+type MapObjectValuesTransformer = (value: any, key: string) => any;
+
+/**
+ * Returns a new object where all values have been transformed.
+ * Similar to https://lodash.com/docs/#mapValues or
+ * https://underscorejs.org/#mapObject
```

Comment:
Hmm, did we copy the API signature or the implementation, too? If latter, we need to attribute this code to them, right?

Reply:
The implementation is not borrowed, but I've updated the comment in the code to note that the API signature was intended to be similar to popular libraries' equivalents for familiarity.